control_plane/src/local_env.rs

```diff
@@ -63,6 +63,34 @@ pub struct LocalEnv {
     #[serde(default)]
     pub broker_endpoints: Option<String>,
 
+    // s3 bucket name for SK WAL Backup
+    #[serde(default)]
+    pub wal_backup_s3_bucket: Option<String>,
```

Comment:
Feels a bit odd that pageserver managed to have its buckets enabled on stating without the control_plane adjustments.
I'd hide those at least into `SafekeeperConf` at least and prepared some common struct just in case pageserver would also need such params here?